### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Add - WhishList

Preview - https://deploy-preview-3--vishnu-store.netlify.app/pages/wishlist.html

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
my first observation is change the title of the wishlist page from cart to wishlist

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://some-ui-components.netlify.app/export.css"
+    />
+    <link rel="stylesheet" href="../stylesheets/wishlist.css" />
+
+    <title>Cart</title>
```
Comment (Dev-AkshatGupta) [inline pages/wishlist.html:13]: Change this to wishlist

**Review by Dev-AkshatGupta — APPROVED** [review]
Else it looks great to me to be merged

File: pages/wishlist.html (hunk 1 above)
Comment (Vishnu-Aithal) [reply pages/wishlist.html:13]: Will Do It 😅
Thanks For the Review

**Review by ravikumar1002 — APPROVED** [review]

**Pull request merged**